app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,68 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
```

Comment:
does this page show a list of tweets?

I would imagine you would want something here that sets an instance variable for all tweets.

```ruby
@twets = Tweet.all 
``` 

?